archetypes/catalog/pom.xml

```diff
@@ -67,4 +67,95 @@
+                                                                 version="${project.version}"
+                                                                 type="zip"
+                                                                 target="."/>
+                                            </unpack-artifacts>
+                                            <templates>
+                                                <template source="cli-metadata.properties.mustache"
+                                                          target="metadata.properties">
+                                                    <variables>
+                                                        <variable name="buildtoolsVersion"
+                                                                  value="${build.tools.version}"/>
+                                                        <variable name="cliVersion"
+                                                                  value="${version.plugin.helidon-build-tools}"/>
+                                                    </variables>
+                                                </template>
+                                            </templates>
+                                        </archive>
+                                    </archives>
+                                    <files>
+                                        <file target="cli-data/latest">${project.version}</file>
+                                    </files>
+                                </directory>
+                            </directories>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.soebes.maven.plugins</groupId>
+                <artifactId>echo-maven-plugin</artifactId>
+                <version>0.4.0</version>
+                <executions>
+                    <execution>
+                        <phase>install</phase>
+                        <goals>
+                            <goal>echo</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <echos>
+                        <echo> |</echo>
+                        <echo> |                      /')    </echo>
+                        <echo> |               ////  /' )'   </echo>
+                        <echo> |              @   \/'  )'    </echo>
+                        <echo> |  { note }   &lt; (  (_...)' </echo>
+                        <echo> |               \      )      </echo>
+                        <echo> |                \,,,,/       </echo>
+                        <echo> |                 _|_         </echo>
+                        <echo> |</echo>
+                        <echo> |  You must add the following arguments to the CLI to use these snapshot archetypes:</echo>
```

Comment:
There is nothing to do unless one wants to test the archetypes. Can you change the beginning of this sentence to provide some context ? E.g. `You can test the archetypes locally with the Helidon CLI using the following arguments:`

Also, the version is not always snapshot. E.g. if you build a tag.